datalad_osf/remote.py

```diff
@@ -118,7 +99,7 @@ def transfer_store(self, key, filename):
             # but you can create_file("a/b/c/d.bin"), and in fact you *cannot* create_folder("c").create_file("d.bin")
             # TODO: patch osfclient to be more intuitive.
 
-            osf_makedirs(self.project.storage(), self.path, exist_ok=True)
+            self._osf_makedirs(self.project.storage(), self.path, exist_ok=True)
```

Comment:
Does this really need to be a method?

It should live in osf. It's not part of the remote really, imo.

Reply:
I never should have committed `annex.info('making')`, I was just debugging stuff.

Reply:
Sure, feel free to make a PR against the osfclient fork, @kousu adn then have it refactored here.